usaspending_api/download/filestreaming/download_generation.py

```diff
@@ -96,7 +100,7 @@ def generate_download(download_job):
         raise Exception(download_job.error_message) from e
     finally:
         # Remove working directory
-        if os.path.exists(working_dir):
+        if working_dir and os.path.exists(working_dir):
```

Comment:
Why is this check needed? It seems like working_dir is still always being set.

Reply:
Possible that the try block could fail before `working_dir` is set and then it would have a value of None in the Finally block.

Reply:
Yep, that's it @sethstoudenmier